RELEASENOTES/v2.0.0.alpha-4.MD

```diff
@@ -0,0 +1,58 @@
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v0.10.0-alpha.4)](https://github.com/notaryproject/notation/releases/tag/v0.10.0-alpha.4)***   (Updated in this release) 
+## Goal of the release
+- Update Notary v2 specifications 
+    - Community can use the updated specifications to develop their own implementation of Notary v2 in language/platforms of their choice, and/or contribute to Notation client development.
+    - Signature format (JWS) for signing and verification in RC-1.
+    - Updates on use of plugins for signing and verification. Plugins let vendors and users integrate base Notation client with their choice of Key Vaults and PKI.
+ - Update "Notation" (CLI client) implementation to match the latest specifications. With the new client, users can
+  - Besides the features from Alpha-3, this releases adds support for "self signed certificates".
+
+*We do not recommend this release for use in production environment. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). We suggest using it for testing and providing feedback to the Notary v2 working group only*
+
+## List of feature/capabilities added with this release
+For detailed notes on each of the individual repositories (NotaryProject, Notation (for reference CLI implementation) and Notation libraries) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notary v2 specifications
+Updated Trust Policy, Trust Store,  Signature Verification and Signing workflows along with extensibility mechanisms via Notation plugins.
+- A final JWS based [signature envelope format.](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md)
+- Updated [Signature specfication](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md) with extensibility provisions for additional signature envelope format.
+- Updated [signing and verification workflows](https://github.com/notaryproject/notaryproject/blob/main/signing-and-verification-workflow.md)
+- Introducing a new [Signing scheme](https://github.com/notaryproject/notaryproject/blob/main/signing-scheme.md) to integrate Notation Client with Signing Services
+- Updated  [trust store and trust policy](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md) specification
+
+### Notation CLI Client
+Notation is a standalone CLI client based on Notary v2 specs. It supports signing, verifying, and storing signatures in registries supporting signatures as references to images. Refer description below for Notation libraries on how to configure Trust store and trust policy to use with Notation client.
+
+In this release of Notation command, the following CLI commands are supported. Refer (TBD - Getting started here)
+- notation sign  
```

Comment:
Can we mention the COSE envelope support in the notation sign? This has been included in the CLI and library.